python/doc/examples/meta_modeling/general_purpose_metamodels/plot_general_linear_model.py

```diff
@@ -63,14 +63,14 @@
 xmax = 5.0
 x1value = 0.5
 parametricModelGraph = ot.ParametricFunction(model, [0], [x1value]).draw(xmin, xmax)
-parametricModelGraph.setColors(["red"])
 graphMetamodel = ot.ParametricFunction(responseSurface, [0], [x1value]).draw(xmin, xmax)
-graphMetamodel.setColors(["blue"])
```

Comment:
Unsure about this deletion, were the colors not specified in order to override the default in this particular case ?